src/components/settings/TranslatorMode.svelte

```diff
@@ -38,8 +39,14 @@
     {#if $doAutoPrefix}
       <TextField
         bind:value={$autoPrefixTag}
-        label="Tag to prepend ($filterLang is replaced by your filter language)"
+        label="Tag to prepend ($filterLang is replaced by the language selected below)"
       />
+      <Dropdown
+        store={prefixTagReplacementLanguage}
+        items={languagesInfo.map((language) => language.lang)}
+      >
+        Auto Prefix Tag
+      </Dropdown>
```

Comment:
Should add a `name` to the dropdown, right now it looks awkwardly empty:

![image](https://user-images.githubusercontent.com/20059164/150675815-45211b08-84d8-4ee5-87c6-ce972e22b5fe.png)

Reply:
Not 100% sure on how to phrase it so check it out @ChrRubin

Reply:
Something like "Prefix language" should be straightforward enough I guess